src/primality_check.jl

```diff
@@ -36,7 +36,11 @@ function check_primality(polys::Dict{fmpq_mpoly, fmpq_mpoly}, extra_relations::A
 end
 
 #------------------------------------------------------------------------------
+"""
+    check_primality(polys::Dict{fmpq_mpoly, fmpq_mpoly}) 
 
+Check if the ideal generated by `polys` is prime.
```

Comment:
@pogudingleb could you please verify if this is a correct statement for this function?

Reply:
@iliailmer 
It is a bit tricker.
The input dictionary is of the form variable => polynomial such that the variable occurs only in the polynomial. The function determines whether the ideal generated by the polynomials and saturated at the leading coefficient with respect to the corresponding variables is prime (also important that this is about primality over QQ). Does this make more sense now?

The version with `extra_relations` just adds extra polynomials to the generators (not affecting the saturation)